clients/ctl/admin-ui/src/features/system/ManualSystemFlow.tsx

```diff
@@ -1,6 +1,14 @@
-import { Box, Button, Grid, Stack, Text } from "@fidesui/react";
+import { Button, Grid, GridItem, Stack, Text } from "@fidesui/react";
+import { useRouter } from "next/router";
 import { useState } from "react";
 
+import { System } from "~/types/api";
+
+import DescribeSystemsForm from "../config-wizard/DescribeSystemsForm";
+import PrivacyDeclarationForm from "../config-wizard/PrivacyDeclarationForm";
+import ReviewSystemForm from "../config-wizard/ReviewSystemForm";
+import SuccessPage from "../config-wizard/SuccessPage";
```

Comment:
I like how you've extracted the state management from these components so that they can be reused like this. These imports make me think the dependence should probably go the other way, though: the config wizard using the components from the system feature. Could be a TODO to move them.

Reply:
yeah I was starting to think that as I was working on this. felt it even more in https://github.com/ethyca/fides/pull/1082. I'll make a ticket to refactor after this PR and https://github.com/ethyca/fides/pull/1082 are in 👍

Reply:
🎫  https://github.com/ethyca/fides/issues/1084